components/molecules/TextButtonField.js

```diff
@@ -0,0 +1,89 @@
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+
+/**
+ *  component
+ */
+export function TextButtonField(props) {
+  return (
+    <div className={props.className + " mb-4"}>
+      <h1>{props.title}</h1>
+
+      {props.text.map((paragraph, key) => {
```

Comment:
Follow the example of setting markdown generated HTML from the NextJS tutorial. Array of text will not be passed in. Just HTML that needs to be set